bin/i18n/sync-out.rb

```diff
@@ -164,15 +151,15 @@ def self.restore_redacted_files
               plugins << 'visualCodeBlock'
               plugins << 'link'
               plugins << 'resourceLink'
-            elsif app_types_with_link.include?(File.basename(original_path, '.json'))
-              plugins << 'link'
+            elsif I18n::Resources::Apps::Labs::REDACTABLE.include?(File.basename(original_path, '.json'))
```

Comment:
Hopefully when all modules are moved out of the sync-out this long if statement will be removed.

Reply:
That's the plan